### PR TITLE
Dockerfile: upgrading Newrelic to 7.2.0.191

### PR DESCRIPTION
**Pull request by bryanlatten**
https://docs.newrelic.com/docs/release-notes/agent-release-notes/php-release-notes/php-agent-720191

- Had to "upgrade" alpine packages that are no longer supported pre-PHP 7.1, building from source

### CODE REVIEW

**Comment by bossjones** [comment]
👍

**Comment by mikeklein13** [comment]
👍

**Comment by mannytoledo** [comment]
👍

File: Dockerfile-alpine [hunk 1]
```diff
@@ -49,8 +49,8 @@ RUN apk update && \
       php7-json \
       php7-mcrypt \
       php7-mbstring \
-      php7-msgpack@edge \
-      php7-memcached@edge \
+      # php7-msgpack@edge \
```
Comment (bryanlatten) [inline Dockerfile-alpine:52]: no longer available for non 7.1 builds, must build from source below...

File: container/root/tests/php-fpm/alpine.goss.yaml [hunk 2]
```diff
@@ -52,10 +52,10 @@ package:
     installed: true
   php7-mbstring:
     installed: true
-  php7-msgpack:
-    installed: true
-  php7-memcached:
-    installed: true
+  # php7-msgpack:
```
Comment (bryanlatten) [inline container/root/tests/php-fpm/alpine.goss.yaml:55]: no longer installed as packages, but are still tested via base goss tests

File: Dockerfile-alpine [hunk 3]
```diff
@@ -118,20 +118,30 @@ RUN cp /etc/php/7.0/mods-available/* $CONF_PHPMODS && \
...
-    # Install new PHP7-stable version of Yaml \
+
     pecl install yaml-2.0.0 && \
     echo ";extension=yaml.so" > $CONF_PHPMODS/yaml.ini && \
-    # Install new PHP7-stable version of Redis \
+
     pecl install redis-3.1.1 && \
     echo ";extension=redis.so" > $CONF_PHPMODS/redis.ini && \
+
+    pecl install msgpack-2.0.2 && \
```
Comment (mannytoledo) [inline Dockerfile-alpine:139]: Still using a package library, so I guess that helps

**Comment by mannytoledo** [comment]
👍

**Comment by bossjones** [comment]
👍

**Pull request merged**